sdk/src/client/api/types.rs

```diff
@@ -74,6 +74,15 @@ impl TryFromDto<PreparedTransactionDataDto> for PreparedTransactionData {
     }
 }
 
+impl Serialize for PreparedTransactionData {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        PreparedTransactionDataDto::from(self).serialize(serializer)
```

Comment:
We still need the DTO because we can't deserialize ?

Reply:
Yeah, unfortunately